### DISCUSSION TITLE
Error while deploying to testnet:

### DISCUSSION THREAD

**Discussion by Tarun-k-Gupta** [Cyfrin/foundry-full-course-cu#1233] [General]
forge script script/DeploySimpleStorage.s.sol --rpc-url $GOERLI_RPC_URL --private-key $PVT_KEY --broadcast
Error: 
Deserialization Error: expected value at line 1 column 1. Response: project id required in the url.

I am using the following rpc url: https://goerli.infura.io/v3/

**Comment by harissatti** [comment]
same issue anyone know the solution?

**Reply by EngrPips** [reply]
Are you using `goerli` url as well?

**Comment by MostPow3rful** [comment]
checkout here :
https://chainlist.org/?search=goerli&testnets=true
